bitrise.yml

```diff
@@ -101,7 +112,7 @@ workflows:
         - content: |-
             #!/usr/bin/env bash
             set -e
-            cd Shopist && make
+            echo 'DATADOG_CLIENT_TOKEN="fake-client-token-for-shopist"' > xcconfigs/Datadog.local.xcconfig
```

Comment:
ℹ️ Makefile for Shopist can take `client_token` as an argument, `make Shopist secret="fake-client-token"` should work as well

Reply:
Yay, saw it, but I wanted to limit number of `xcconfig` files, so the `./Shopist/shopist-secrets.local.xcconfig` was removed in this PR (together with Shopist's `Makefile`). Now all projects in the repository read the config from `xcconfigs/Datadog.local.xcconfig`.